bedrock/gigabit/templates/gigabit/apply.html

```diff
@@ -152,6 +152,9 @@ <h1 class="large center">{{ _('Apply') }}</h1>
             {% trans url_startupCompete='//gigabitfund2016.startupcompete.co/' %}
               Register on our <a href='{{ url_startupCompete }}'>StartupCompete site</a> and submit your application for review.
             {% endtrans %}
+            {% trans url_video='https://vimeo.com/180975946' %}
```

Comment:
According to https://github.com/mozilla/learning.mozilla.org/issues/2174 this should be added as a sixth item to the list, so it should be within it's own `<li>`